jazz_codeq/utils.js

```diff
@@ -151,7 +150,7 @@ function getJazzToken(config) {
 		
 		logger.debug("Getting token for calling Environment API...");
 		request(svcPayload, (error, response, body) => {
-			if (response.statusCode === 200 && body && body.data) {
+			if (response.statusCode === 200 && body) {
```

Comment:
Why did we remove this check -  && body.data?

Reply:
Its coming as string. so doing body.data gives undefined.

Reply:
Updated